script/configMachines.sh

```diff
@@ -18,17 +18,16 @@ AllCookies=`cat /root/basho_bench1/basho_bench/script/allcookiesfull`
 echo All nodes "$AllNodes"
 mkdir -p logs/"$JobId"
 
-File1="./antidote/rel/vars.config"
+File1="./antidote/rel/vars/dev_vars.config.src"
```

Comment:
Actually, I'm not sure if this is the file that needs to be changed. I see there's another configuration file in `antidote/config/vars.config` with the same information.